src/renderer/components/+network-ingresses/ingress-details.tsx

```diff
@@ -141,3 +139,11 @@ kubeObjectDetailRegistry.add({
     Details: (props) => <IngressDetails {...props} />
   }
 })
+kubeObjectDetailRegistry.add({
+  kind: "Ingress",
+  apiVersions: ["extensions/v1beta1"],
```

Comment:
should we register also `networking.k8s.io/v1`?

Reply:
Isn't that out-of-scope of this PR?

Reply:
probably since we are using `extensions/v1beta1` in api too.

Reply:
`networking.k8s.io/v1` should be done in a separate PR, it most likely will affect also how UI should be rendered.

Reply:
Created an issue: https://github.com/lensapp/lens/issues/1347